src/layout/MasterLayout.cpp

```diff
@@ -339,6 +339,27 @@ void CHyprMasterLayout::calculateWorkspace(const int& ws) {
     if (!PMASTERNODE)
         return;
 
+    // dynamic workspace rules
+    const auto  WORKSPACERULES = g_pConfigManager->getWorkspaceRulesFor(g_pCompositor->getWorkspaceByID(ws));
+    std::string orientationForWs;
+
+    for (auto& wsRule : WORKSPACERULES) {
+        if (wsRule.layoutopts.contains("orientation"))
+            orientationForWs = wsRule.layoutopts.at("orientation");
+    }
+
+    if (orientationForWs == "top") {
```

Comment:
all this time and we still forget no {} around short ifs? :P

Reply:
you got me. i actually hid a backdoor in the {}s.
anyway deleted now